repository/s.json

```diff
@@ -2565,6 +2565,19 @@
 				}
 			]
 		},
+		{
+			"name": "SnailSVN_Helper",
+			"description": "This is a SnailSVN Helper for Sublime",
```

Comment:
That much is clear from the name, but what does it do?

Reply:
Also, you can remove this and instead simply fill in a description on your GitHub repo. That will be used instead.